plaso/parsers/text_parser.py

```diff
@@ -179,6 +180,33 @@ def __init__(self):
     # TODO: self._line_structures is a work-around and this needs
     # a structural fix.
     self._line_structures = list(self.LINE_STRUCTURES)
+    self._parser_mediator = None
+
+    codecs.register_error('text_parser_handler', self._EncodingErrorHandler)
+
+  def _EncodingErrorHandler(self, exception):
+    """Encoding error handler.
+
+    Args:
+      exception [UnicodeDecodeError]: exception.
+
+    Returns:
+      tuple[str, int]: replacement string and number of bytes to skip.
```

Comment:
The docs for [register_error() ](https://docs.python.org/3/library/codecs.html#codecs.register_error) say that the int here is where encoding should continue, not the number of bytes that should be skipped. Please correct the docstring here.

Reply:
changed